parquet-hadoop/src/main/java/org/apache/parquet/crypto/AesCipher.java

```diff
@@ -78,11 +78,22 @@
     if (rowGroupOrdinal < 0) {
       throw new IllegalArgumentException("Wrong row group ordinal: " + rowGroupOrdinal);
     }
-    byte[] rowGroupOrdinalBytes = shortToBytesLE(rowGroupOrdinal);
+    short shortRGOrdinal = (short) rowGroupOrdinal;
+    if (shortRGOrdinal != rowGroupOrdinal) {
+      throw new ParquetCryptoRuntimeException("Encrypted parquet files can't have "
+          + "more than Short.MAX_VALUE row groups: " + rowGroupOrdinal);
```

Comment:
I think, writing the actual value instead of referencing a java constant is more informative.